src/photos/locales/en.json

```diff
@@ -125,6 +125,7 @@
         "email": "email",
         "emailPlaceholder": "Enter email address or name of your recipient",
         "send": "Send",
+        "genericSuccess": "You sent an invite to %{count} contacts.",
```

Comment:
This translation probably needs a singular form as well, if there is only one contact?

Reply:
No, because it's only displayed when there are many contacts.